x-pack/plugins/observability_solution/observability_ai_assistant/server/functions/summarize.ts

```diff
@@ -41,6 +42,11 @@ export function registerSummarizationFunction({
             type: 'boolean',
             description: 'Whether this is a correction for a previous learning.',
           },
+          // type: {
+          //   type: 'string',
+          //   description: `The type can be: "user_instruction" or "contextual". A "user_instruction" entry will be included in the system message if it fits within the token budget and it will be marked as a user instruction. A "contextual" entry will be included as part of the context function response (and not the system message) if it semantically matches the user's prompt.`,
+          //   enum: [KnowledgeBaseType.UserInstruction, KnowledgeBaseType.Contextual],
+          // },
```

Comment:
should we remove this? I can also clean it up if you want to, LMK

Reply:
Yeah I was going to ask you what you thought. In my mind the LLM should never create an entry with type `user_instruction`. Before commenting this out it did sometimes

Reply:
agree, I think we can remove it entirely and hardcode it to `Contextual`.